src/test/people/edit/post.controller.test.js

```diff
@@ -166,4 +166,31 @@ describe('Person Edit Post Controller', () => {
       expect(res.render).to.not.have.been.called;
     });
   });
+  it('should render with people other document type validating the alpha numeric stuff', () => {
```

Comment:
I think you could make the description clearer.  Something like:  'should render with validation messages if "Other" document type is selected and special characters, apostrophes or numbers are present in the given name'

I presume the req.body.firstName is "Given Name" on the form.  Maybe you could leave a comment to indicate this.